crates/astria-composer/src/searcher/mod.rs

```diff
@@ -226,6 +245,39 @@ impl Searcher {
         }
     }
 
+    fn spawn_executor(&mut self) -> tokio::task::JoinHandle<()> {
+        // spawn executor task
+        let handle = tokio::spawn(
```

Comment:
Nit: would prefer to be consistent between `tokio::spawn` or `tokio::task::spawn` and not use both

Reply:
good catch - thanks